### PR TITLE
Part1.2 

### PR DESCRIPTION
**Pull request by Victoria-Moiseienko**

### CODE REVIEW

**Review by matviichuk92 — APPROVED** [review]
good job!

File: src/main/java/com/internet/shop/service/OrderService.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.internet.shop.service;
+
+import com.internet.shop.model.Order;
+import java.util.List;
+
+public interface OrderService {
```
Comment (matviichuk92) [inline src/main/java/com/internet/shop/service/OrderService.java:7]: в  OrderService мають бути наступні методи: 
- Order completeOrder(ShoppingCart shoppingCart);
- List<Order> getUserOrders(Long userId)
- Order get(Long id);
- List<Order> getAll();
- boolean delete(Long id);

Comment (Victoria-Moiseienko) [reply src/main/java/com/internet/shop/service/OrderService.java:7]: Oh. Thank you

**Comment by RomanDubovskyi** [comment]
you have files from first PR, that are not related to current HW, merge your previous PR into master, so they won't be in "Files changed"

**Pull request closed without merging**